lineage_beyond0lte.mk

```diff
@@ -24,11 +24,10 @@ PRODUCT_BRAND := samsung
 PRODUCT_MODEL := SM-G970F
 PRODUCT_MANUFACTURER := samsung
 PRODUCT_PDA_MODEL := G970F
+PRODUCT_PDA_MODEL_VERSION := XXSBFUF3
 PRODUCT_PDA_VERSION := $(PRODUCT_PDA_MODEL)$(PRODUCT_PDA_MODEL_VERSION)
 PRODUCT_GMS_CLIENTID_BASE := android-samsung
 
-PRODUCT_PDA_MODEL_VERSION := XXSBFUF3
-
```

Comment:
I've moved it down so that cherry-picking changes to other devices is easy. The PDA_VERSION should be moved down too.